app/src/main/java/com/sopt/androidstudy/presentation/save/screens/FriendDetailActivity.kt

```diff
@@ -0,0 +1,39 @@
+package com.sopt.androidstudy.presentation.save.screens
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.databinding.DataBindingUtil
+import androidx.lifecycle.ViewModelProvider
+import com.sopt.androidstudy.R
+import com.sopt.androidstudy.data.datasources.FriendDataSources
+import com.sopt.androidstudy.data.model.db.Friend
+import com.sopt.androidstudy.data.model.db.FriendDatabase
+import com.sopt.androidstudy.data.repository.FriendRepositoryImpl
+import com.sopt.androidstudy.databinding.ActivityFriendDetailBinding
+import com.sopt.androidstudy.presentation.save.viewmodels.FriendDetailViewModel
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class FriendDetailActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityFriendDetailBinding
+    private val friendDetailViewModel: FriendDetailViewModel by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        initBindingView()
+    }
+
+
+    private fun initBindingView() {
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_friend_detail)
+        binding.viewModel = friendDetailViewModel
+        binding.lifecycleOwner = this
+        val friend = intent.getParcelableExtra<Friend>("friend")
+        if (friend != null) {
+            friendDetailViewModel.setFriend(friend)
+        }
```

Comment:
if (friend != null) 보다는 ?.let이 더 간지나지 않나요? 😎
```suggestion
        friend?.let {
            friendDetailViewModel.setFriend(friend)
        }
```

개인적으로는 가독성을 위해 가능하면 들여쓰기를 피하는 편,, null인 경우 바로 return하고, 그렇지 않은 경우 다음 줄 코드를 실행하죠
```suggestion
        if (friend == null) return
        friendDetailViewModel.setFriend(friend)
```